### ISSUE TITLE
Feature request: symbol annotations

### ISSUE THREAD

**Issue by AMChalkie** [Oshlack/superTranscript_paper_code#3]
If you're going to provide reference files some model organisms, it would be helpful to annotate the gtf with gene symbols to help users find their genes.

**Comment by Quarkins** [comment]
Good point, at the moment the genes are ENSEMBL annotations since that is what we made the supertranscripts with, but we should be able to extract the gene symbol names too. For the time being you can look up your favourite genes ENSEMBL name to find the genes in IGV.

**Comment by AMChalkie** [comment]
Agreed, and I'll be doing my own symbol/gene mapping for our genomes, but think others might not be so keen on that idea.
